goblin/engine.py

```diff
@@ -43,7 +41,7 @@
     msg = await stream.fetch_data()
     features['threaded_transactions'] = msg.data[0]
 
-    return Engine(url, conn, loop, **features)
+    return Engine(url, conn, loop, force_close=force_close, **features)
```

Comment:
See line note below. I think connections should be closed explicitly by closing the engine. In the case of a driver user, connections should be either closed explicitly with ``close`` or by using a CM